modules/sdk/src/main/scala/org/tessellation/sdk/cli/CollateralConfigOpts.scala

```diff
@@ -0,0 +1,21 @@
+package org.tessellation.sdk.cli
+
+import cats.syntax.either._
+
+import org.tessellation.schema.balance.{Amount, _}
+import org.tessellation.sdk.config.types.CollateralConfig
+
+import com.monovore.decline.Opts
+import com.monovore.decline.refined.refTypeArgument
+import eu.timepit.refined.types.numeric.NonNegLong
+
+object CollateralConfigOpts {
+
+  val opts: Opts[CollateralConfig] = Opts
+    .option[NonNegLong]("collateral", help = "User provided collateral")
```

Comment:
Could you change the help to be more descriptive what the collateral is? for example:
```
help = "Minimum staking amount to run a node"
```